Projects/Chat/Sources/Views/ChatScreenViewModel.swift

```diff
@@ -157,6 +164,8 @@ public class ChatScreenViewModel: ObservableObject {
 
     @MainActor
     func retrySending(message: Message) async {
+        let store: ChatStore = globalPresentableStoreContainer.get()
+        store.send(.clearFailedMessage(message))
```

Comment:
I think this cleaning should be done when the message was sent. So no need for this code, lets add it when the message is sent


here:
```
    private func sendToClient(message: Message) async {
        do {
            let sentMessage = try await chatService.send(message: message)
            let store: ChatStore = globalPresentableStoreContainer.get()
            store.send(.clearFailedMessage(message))
```